docs/reference/extensions/gnome-extension.rst

```diff
@@ -0,0 +1,253 @@
+.. _gnome-extension:
+
+GNOME extension
+===============
+
+The GNOME extension, referred to internally as ``gnome``, helps build snaps that use GTK
+3, GNOME 42 and higher, and GLib. This extension provides many of the components needed
+for general desktop apps, making it useful for a broader set of apps outside of those
+tailored for the GNOME desktop.
+
+This extension is compatible with the core22 and core24 bases.
+
+
+.. _gnome-extension-included-plugs:
+
+Included plugs
+--------------
+
+The extension connects the following snap-wide plugs.
+
+.. collapse:: Included snap-wide plugs
+
+    .. code-block:: yaml
+
+        plugs:
+          - desktop
+          - desktop-legacy
+          - gsettings
+          - opengl
+          - wayland
+          - x11
+          - mount-observe
+          - calendar-service
```

Comment:
After re-reviewing, the document feels extremely sparse without the inclusion of any code prior to the example project.

I feel that the longevity of this doc structure might be dubious. We might yield more useful results if we ran a script to autogenerate the project file diffs and let the user read them for themselves. I don't know how useful it is to partition these lists without any description of what the various keys are doing.

Anyway, this shouldn't be a blocker, but I want it noted somewhere.

Reply:
Created CRAFT-4050 to track this idea.